docs/install/upgrade-installation.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Upgrading your installation"
+weight: 21
+type: "docs"
+---
+
+If you already have Knative installed, upgrade your installation by running the
```

Comment:
I would remove the conditional since you can assume that something should be installed to be able to upgrade it and not say "for the new version" because the subsequent statement says not to simply jump to the latest. Also for clarity, I think we should identify what you can upgrade and specifically "components and plugins". 

Maybe something along the lines of:

"To upgrade your Knative components and plugins, you run the `kubectl apply` command to install the subsequent release...."